tests/serverDescription/serverDescription-getType-001.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+MongoDB\Driver\ServerDescription::getType()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = create_test_manager();
+$server = $manager->selectServer(new MongoDB\Driver\ReadPreference('primary'));
+
+var_dump($server->getServerDescription()->getType());
```

Comment:
If you'd like to improve the expectation, you can assert that type is either a standalone, mongos, or primary (per constants in https://www.php.net/manual/en/class.mongodb-driver-server.php). Those are the only things we'd expect given the read preference above.